Automattic/wp-calypso
# Jetpack Backup: i4 backup states

### Changes proposed in this Pull Request

This PR was a branch off of `'improve/backup-feed`.

It implements the i4 iteration of the _no-backups-yet_ card, _backup-scheduled_ card, _backup-failed_ card, _no-backups-on-selected-date_ status cards.


### Testing instructions

Feature flag = `?flags=jetpack/backup-simplified-screens-i4`

See the link to the Figma design mockups in the Asana card: 1197351014149633-as-1197888024354844/f

1. Run this PR with a site that has backups enabled.
    - Note: If you have an expired JN site (+7 days old) that had Backups enabled, the site should contain the various backup states (backup scheduled, no-backup-on-selected date, backup-failed) on various selected days. To test _no-backups-yet_ (First Backup will be ready soon), you may need to create a new JN site, & purchase backup. Or you can also manually set variables to trigger the various states.
2. Visit /backup/:site/?flags=jetpack/backup-simplified-screens-i4
3. Check the various backup states and verify that they look like the Figma mockups and that they function properly.

### Screenshots

![Screenshot on 2020-10-21 at 09-30-23](https://user-images.githubusercontent.com/11078128/96751298-db8b9900-1381-11eb-9d3c-d25d5146088e.png)

![Screenshot on 2020-10-21 at 09-31-50](https://user-images.githubusercontent.com/11078128/96751324-e2b2a700-1381-11eb-8cbd-36fddf9a5266.png)

![Screenshot on 2020-10-21 at 09-32-26](https://user-images.githubusercontent.com/11078128/96751336-e6462e00-1381-11eb-820d-bb656c96cbc5.png)

![Screenshot on 2020-10-21 at 09-32-55](https://user-images.githubusercontent.com/11078128/96751346-ea724b80-1381-11eb-9fbe-1c4cc0454737.png)

**Environments to test:**

- Calypso vs. Jetpack cloud
- Desktop vs. mobile

Fixes 1197351014149633-as-1197888024354844/f
